scripts/test/system.sh

```diff
@@ -47,6 +47,43 @@ function test_subctl_gather() {
     echo "::endgroup::"
 }
 
+function check_service_exported() {
+    status=$(kubectl get serviceexport nginx-demo --context "$cluster" -n default -o=jsonpath='{.status.conditions[?(@.type=="Valid")].status}')
+    if [[ "$status" == "True" ]]; then
+        return 0
+    fi
+
+    c=$(kubectl get serviceexport nginx-demo --context "$cluster" -n default -o=jsonpath='{.status.conditions[?(@.type=="Valid")]}')
+    echo "Service not exported: $c"
+
+    return 1
+}
+
+function export_service() {
```

Comment:
```suggestion
function export_service() {
    echo "::group::Running & validating 'subctl export service'"
```

Makes it easier to read the output of the job